packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -391,6 +407,50 @@ export async function handleConfirmation(args: {
       });
     });
 
+    scheduleTriggerPromises.push(
+      ...subscribersHostsNoShowStarted.map((webhook) => {
+        if (booking?.startTime && webhook.time && webhook.timeUnit) {
+          const scheduledAt = dayjs(booking.startTime)
+            .add(webhook.time, webhook.timeUnit.toLowerCase() as dayjs.ManipulateType)
+            .toDate();
+          return tasker.create(
+            "triggerHostNoShowWebhook",
+            {
+              triggerEvent: WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
+              bookingId: booking.id,
+              // Prevents null values from being serialized
+              webhook: { ...webhook, time: webhook.time, timeUnit: webhook.timeUnit },
+            },
+            { scheduledAt }
+          );
+        }
+        return Promise.resolve();
```

Comment:
create task for triggerHostNoShowWebhook and triggerGuestNoShowWebhook

Reply:
Can we extract this logic and use it both on handleNewBooking and here? To ensure and prevent logic drifts?

Reply:
done